templates/components/form/fields_macros.html.twig

```diff
@@ -878,8 +878,12 @@
    {% set locked_mark = '' %}
    {% if options.locked %}
       {% set locked_mark %}
+        {% set locked_title %}{{ __('Field will not be updated from inventory') }}{% endset %}
+        {% if options.locked_value is not empty %}
```

Comment:
`locked_value: null` should be added in options definition, on line 859.

Reply:
done